kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -350,70 +432,96 @@ def find_vm_use_falut_nic_without_virsh(self, falut_nic):
+    def _get_current_down_business_nics(self, nics):
+        down_nics = []
+        for nic in nics:
             try:
-                ip = iproute.query_links(new_nic)
+                ip = iproute.query_links(nic)
                 if ip[0].state == 'DOWN':
-                    self.falut_nic_count[new_nic] += 1
-                else:
-                    self.falut_nic_count[new_nic] = 0
+                    down_nics.append(nic)
             except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)